query-engine/connectors/sql-query-connector/src/row.rs

```diff
@@ -34,11 +34,11 @@ impl ToSqlRow for ResultRow {
         let mut row = SqlRow::default();
         let row_width = idents.len();
         for (i, p_value) in self.into_iter().enumerate().take(row_width) {
-            let pv = match idents[i] {
+            let pv = match &idents[i] {
                 (type_identifier, FieldArity::List) => match p_value {
                     ParameterizedValue::Array(l) => l
                         .into_iter()
-                        .map(|p_value| row_value_to_prisma_value(p_value, type_identifier))
+                        .map(|p_value| row_value_to_prisma_value(p_value, type_identifier.clone()))
```

Comment:
Instead of cloning (due to the removed `Copy` implementation). Why not sending the `type_identifier` as a reference to the `row_value_to_prisma_value`?

Reply:
Already adressed after the review with @tomhoule